components/label/label.scss

```diff
@@ -0,0 +1,31 @@
+.label {
+    padding: 0.2em 0.5em 0.3em;
+    border-radius: 4px;
+    font-size: 10px;
+
+    &-danger {
+        background-color: var(--dnd-indicator);
+    }
+
+    &-success {
+        background-color: var(--online-indicator);
+    }
+
+    &-warning {
+        background-color: var(--mention-highlight-bg);
+    }
+
+    &-info {
+        background-color: var(--link-color);
+    }
+
+    &-default {
+        background-color: var(--center-channel-color);
+        color: var(--center-channel-bg);
+    }
+
+    &-primary {
+        background-color: var(--sidedar-header-bg);
+        color: var(--center-channel-color);
+    }
```

Comment:
@matthewbirtch can you help verify/review these colors for the label component?
I had to create one for the drafts header, and I took the liberty to add colors for other ones as well, 
but they might be off.

PS: I'm also OK with deleting the other colors for now and adding only when necessary.

Reply:
> Thanks! Can we change the following?
> 
> ```
> &-warning {
>     background-color: var(--away-indicator);
> }
> ```
> 
> ```
> &-info {
>     background-color: var(--sidebar-text-active-border);
> }
> ```
> 
> ```
> &-primary {
>     background-color: var(--button-bg);
> }
> ```

They look great thx!

Reply:
@koox00 this is something we'll want to consider for the design system. I'll create a ticket for UX to review and potentially include this in the component when UI Platform builds it out